gtpv1/tunnel_linux.go

```diff
@@ -24,18 +24,18 @@ const (
 // EnableKernelGTP enables Linux Kernel GTP-U.
 // Note that this removes all the existing userland tunnels, and cannot be disabled while
 // the program is working (at least at this moment).
-//
```

Comment:
Please don’t remove this so that godoc parser can regard the whole comment as a documentation.

Reply:
<img width="328" alt="Screen Shot 2022-09-01 at 4 12 51 PM" src="https://user-images.githubusercontent.com/47399469/187922476-acb10ce7-8349-4f7c-b106-4f44f74e9482.png">

Tried to fix a linter error related to syntax complexity

Reply:
OK, noted as an issue 👍